support-frontend/assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.scss

```diff
@@ -1,124 +1,121 @@
-// -----  gu-sass ----- //
-
-.thank-you__container .header{
-  @include mq($from: mobile, $until: tablet) {
-    background-color: gu-colour(highlight-main);
+@import './ContributionThankYouSetPassword';
+
+.thank-you__container {
+  display: flex;
+  flex-flow: column-reverse nowrap;
+  justify-content: flex-start;
+  margin: auto;
+  max-width: 1015px;
+
+  @include mq($from: tablet) {
+    flex-flow: row nowrap;
+    justify-content: space-around;
   }
-}
 
-.confirmation {
-  border-bottom: 1px solid gu-colour(neutral-86);
-  font-weight: 300;
-  margin: 0 -10px;
-  padding: 0 10px;
-}
-
-.confirmation__title {
-  font-weight: 900;
-  font-size: 1;
-  margin: ($gu-v-spacing / 3) 0;
-}
+  @include mq($from: leftCol) {
+    justify-content: center;
+  }
 
-.confirmation__message {
-  font: 300 22px/1.2 $gu-headline;
-  margin: ($gu-v-spacing / 4) 0 $gu-v-spacing * 2;
-}
+  .gu-content__blurb.gu-content__blurb--thank-you {
+    display: block;
+    border-top: none;
 
-.confirmation__message--direct-debit {
-  margin-left: -10px;
-  margin-right: -10px;
-  padding: 0 10px $gu-v-spacing;
-  border-bottom: 1px solid gu-colour(neutral-86);
-}
+    @include mq($from: tablet) {
+      border-top: 10px solid gu-colour(neutral-7);
+    }
+  }
 
-.set-password__title {
-  font-family: $gu-text-egyptian-web;
-  letter-spacing: -0.5px;
-  font-weight: 900;
-  font-size: 17px;
-  margin: ($gu-v-spacing / 3) 0;
-}
+  .gu-content__blurb-header {
+    margin: 0;
+    font-family: $gu-text-egyptian-web;
+    letter-spacing: -0.5px;
+    font-weight: 900;
+    font-size: 17px;
+    line-height: 22px;
```

Comment:
`  @include gu-fontset-body-large;`

Reply:
I just tried this and it's a different typeface but I'm adding a "refactor typeface styles using mixins" to TBD